### ISSUE TITLE
Stop/Cancel scrollTo animation

### ISSUE THREAD

**Issue by madprops** [idiotWu/smooth-scrollbar#168]
I'm on Chrome using 8.3.0 (latest as of now)

I'm making a chat application where I want to make an animated scroll to the bottom on each new message.

I'm using something like this:

`			chat_scrollbar.scrollTo(0, y, d,
			{
				callback: function()
				{
					check_scrollers()
				}
			})
`

Which works. But the thing is if a new message arrives while it's animating it messes everything up. 

What I did when I used Jquery's animate was to use .stop() to stop old animations. 

Is there a way to do this with smooth-scrollbar?

**Comment by idiotWu** [comment]
Oh, I just forgot that...

Since I have no time these days, I'd suggest you use [`setMomentum`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbarsetmomentum) method instead:

```js
scrollbar.setMomentum(0, scrollbar.limit.y - scrollbar.offsetTop)
```

**Comment by idiotWu** [comment]
Fixed in v8.3.1

**Comment by andy-hook** [comment]
Hi @idiotWu 
Is there a flag to revert this behaviour? I'd like to prevent a user from cancelling a scroll animation if it's in progress.
